packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCardRow.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC, ReactNode } from 'react';
+import { Text } from '@deriv/quill-design';
+
+type TJurisdictionCardRowProps = {
+    description?: ReactNode | string;
```

Comment:
Dont think we need string here since ReactNode already has string in it. Typescript is smart enough to infer